xbitinfo/xbitinfo.py

```diff
@@ -376,7 +376,102 @@ def load_bitinformation(label):
         raise FileNotFoundError(f"No bitinformation could be found at {label+'.json'}")
 
 
-def get_keepbits(info_per_bit, inflevel=0.99):
+def get_cdf_without_artificial_information(
+    info_per_bit, bitdim, threshold, tolerance, bit_vars
+):
+    """
+    Calculate a Cumulative Distribution Function (CDF) with artificial information removal.
+
+    This function calculates a modified CDF for a given set of bit information and variable dimensions,
+    removing artificial information while preserving the desired threshold of information content.
+
+    Parameters:
+    -----------
+    info_per_bit : :py:class: 'xarray.Dataset'
+        Information content of each bit. This is the output from :py:func:`xbitinfo.xbitinfo.get_bitinformation`.
+    bitdim : str
+        The dimension representing the bit information.
+    threshold : float
+        Determines the percentage of total information above which the keepbits should lie.
```

Comment:
How about:
```suggestion
        Minimum cumulative sum of information content before artificial information filter is applied.
```
Percentage would actually be for the range 0 to 100, but you are rather using a fraction here.